### PR TITLE
Refactor RosTimer to extend TimerAction

### PR DESCRIPTION
**Pull request by rebecca-butler**
This PR relies on:
- [x] ros2/launch#512

Signed-off-by: Rebecca Butler rebecca@openrobotics.org

### CODE REVIEW

File: launch_ros/launch_ros/actions/ros_timer.py [hunk 1]
```diff
@@ -72,140 +62,51 @@ def __init__(
...
-        # launch process
-        context.register_event_handler(
-            EventHandler(
-                matcher=lambda event: is_a_subclass(event, Shutdown),
-                entities=OpaqueFunction(function=lambda context: self.cancel())
-            )
-        )
-
+        super().execute(context)
         return None
```
Comment (jacobperron) [inline launch_ros/launch_ros/actions/ros_timer.py:112]:
```suggestion
        return super().execute(context)
```

**Review by jacobperron — COMMENTED** [review]
LGTM

One minor suggestion. Also, see my other comment in the connected PR.

File: launch_ros/launch_ros/actions/ros_timer.py (hunk 1 above)
Comment (rebecca-butler) [reply launch_ros/launch_ros/actions/ros_timer.py]: Done.

**Review by jacobperron — APPROVED** [review]
I would first merge the connected PR, then we can make a release into Rolling before merging this PR.

**Comment by rebecca-butler** [comment]
@ros-pull-request-builder retest this please

**Pull request merged**